src/test/java/teammates/test/cases/datatransfer/AdminEmailAttributesTest.java

```diff
@@ -0,0 +1,254 @@
+package teammates.test.cases.datatransfer;
+
+import static teammates.common.util.FieldValidator.EMAIL_CONTENT_ERROR_MESSAGE;
+import static teammates.common.util.FieldValidator.EMAIL_CONTENT_FIELD_NAME;
+import static teammates.common.util.FieldValidator.EMAIL_SUBJECT_FIELD_NAME;
+import static teammates.common.util.FieldValidator.EMAIL_SUBJECT_MAX_LENGTH;
+import static teammates.common.util.FieldValidator.INVALID_NAME_ERROR_MESSAGE;
+import static teammates.common.util.FieldValidator.REASON_CONTAINS_INVALID_CHAR;
+import static teammates.common.util.FieldValidator.REASON_EMPTY;
+import static teammates.common.util.FieldValidator.REASON_START_WITH_NON_ALPHANUMERIC_CHAR;
+import static teammates.common.util.FieldValidator.REASON_TOO_LONG;
+import static teammates.common.util.FieldValidator.REGEX_NAME;
+import static teammates.common.util.FieldValidator.SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE;
+import static teammates.test.driver.StringHelperExtension.generateStringOfLength;
+
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.google.appengine.api.datastore.Text;
+
+import teammates.common.datatransfer.attributes.AdminEmailAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.FieldValidator;
+import teammates.common.util.StringHelper;
+import teammates.common.util.TimeHelper;
+import teammates.storage.entity.AdminEmail;
+
+/**
+ * SUT: {@link AdminEmailAttributes}.
+ */
+public class AdminEmailAttributesTest extends BaseAttributesTest {
+
+    private FieldValidator fieldValidator = new FieldValidator();
+    private AdminEmailAttributes adminEmailAttributes;
```

Comment:
The way I see it, it looks like this attribute can be assigned directly during initialisation (no need for `classSetup`) if it is put at the bottom of all other fields.